java/build.gradle

```
@@ -20,25 +20,25 @@ def slf4jVersion = '1.7.36'
 def log4jVersion = '2.19.0'
 
 dependencies {
-    compile group: 'org.slf4j', name: 'slf4j-api', version: "${slf4jVersion}"
-    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: "${log4jVersion}"
-    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j-impl', version: "${log4jVersion}"
-    compile group: 'info.picocli', name: 'picocli', version: '4.0.1'
-    compile group: 'javax.json', name: 'javax.json-api', version: '1.1.4'
-    compile group: 'org.glassfish', name: 'javax.json', version: '1.1.4'
-    compile group: 'com.google.api.grpc', name: 'proto-google-common-protos', version: '1.0.0'
-    compile group: 'com.google.api.grpc', name: 'googleapis-common-protos', version: '0.0.3'
-    compile group: 'io.grpc', name: 'grpc-alts', version: "${grpcVersion}"
-    compile group: 'io.grpc', name: 'grpc-netty', version: "${grpcVersion}"
-    compile group: 'io.grpc', name: 'grpc-protobuf', version: "${grpcVersion}"
-    compile group: 'io.grpc', name: 'grpc-stub', version: "${grpcVersion}"
-    compile group: 'com.google.protobuf', name: 'protobuf-java-util', version: "${protobufVersion}"
-    compile group: 'dnsjava', name: 'dnsjava', version: '2.1.9'
+    implementation group: 'org.slf4j', name: 'slf4j-api', version: "${slf4jVersion}"
```

How about using the short format?
```suggestion
    implementation "org.slf4j:slf4j-api:${slf4jVersion}"
```

---

Thanks for the advise!

I revise it in https://github.com/scalar-labs/scalar-admin/pull/40

Please take a look 🙇